src/styles/styles.js

```diff
@@ -1410,12 +1422,85 @@ function getZoomSizingStyle(isZoomed) {
     };
 }
 
-function getSecondAvatarStyle(parentBGColor) {
+/**
+ * Returns a style with backgroundColor and borderColor set to the same color
+ *
+ * @param {String} backgroundColor
+ * @returns {Object}
+ */
+function getBackgroundAndBorderStyle(backgroundColor) {
+    return {
+        backgroundColor,
+        borderColor: backgroundColor,
+    };
+}
+
+/**
+ * Returns a style with the specified backgroundColor
+ *
+ * @param {String} backgroundColor
+ * @returns {Object}
+ */
+function getBackgroundColorStyle(backgroundColor) {
+    return {
+        backgroundColor,
+    };
+}
+
+/**
+ * @param {Animated.Value} rotate
+ * @param {Animated.Value} backgroundColor
+ * @returns {Object}
+ */
+function getAnimatedFABStyle(rotate, backgroundColor) {
+    return {
+        transform: [{rotate}],
+        backgroundColor,
+    };
+}
+
+/**
+ * @param {Number} width
+ * @param {Number} height
+ * @returns {Object}
+ */
+function getWidthAndHeightStyle(width, height) {
     return {
-        backgroundColor: parentBGColor,
-        borderColor: parentBGColor,
+        width,
+        height,
     };
 }
+
+/**
+ * @param {Number} opacity
+ * @returns {Object}
+ */
+function getOpacityStyle(opacity) {
+    return {opacity};
+}
+
+/**
+ * @param {Object} params
+ * @returns {Object}
+ */
+function getModalPaddingStyles({
```

Comment:
Should this (for now at least) be in the `getModalStyles` file?

Reply:
I thought about this, but decided to leave it here since we should maybe import `getModalStyles` into `styles.js` and then re-export it from there to keep things organized (if we decide that makes sense). 

But mostly, I don't want to have to update all imports of `getModalStyles` to use a named export/import.

Reply:
Okay, makes sense!